debian/changelog

```diff
@@ -1,3 +1,21 @@
+cloud-init (20.4.1-79-g71564dce-0ubuntu1) hirsute; urgency=medium
+
+  * New upstream snapshot.
+    - [VMware] change default max wait time to 15s (#774) [xiaofengw-vmware]
```

Comment:
Should manually we add back Release 20.4.1 since we do document that release bump commit in debian/changelog for every other release we've done?
```suggestion
    - [VMware] change default max wait time to 15s (#774) [xiaofengw-vmware]
```
```suggestion
    - Release 20.4.1
    - [VMware] change default max wait time to 15s (#774) [xiaofengw-vmware]
```

Reply:
And I second my same question/opinion here for discussion as a non-bot person.

Reply:
+1; addressed.

As an aside, it wasn't already included because `new-upstream-snapshot` calls `git log` with `--first-parent`, so ignores the commits from the 20.4.1 side of the history:

```diff
$ diff -u <(git log --first-parent --no-decorate --format=full c06d9c7ecb477582700345e42fed79d350357847..upstream/master | log2dch) <(git log --no-decorate --format=full c06d9c7ecb477582700345e42fed79d350357847..upstream/master | log2dch)      
--- /proc/self/fd/11	2021-01-28 12:56:56.982405784 -0500
+++ /proc/self/fd/15	2021-01-28 12:56:56.982405784 -0500
@@ -11,3 +11,5 @@
     - Adding self to cla signers (#776) [Andrew Bogott]
     - doc: avoid two warnings (#781) [Dan Kenigsberg]
     - Use proper spelling for Red Hat (#778) [Dan Kenigsberg]
+    - Release 20.4.1 (LP: #1911680)
+    - Revert "ssh_util: handle non-default AuthorizedKeysFile config (#586)"
```